src/libecalc/fixtures/case_utils.py

```diff
@@ -3,10 +3,10 @@
 
 from libecalc.fixtures import YamlCase
 from libecalc.infrastructure.file_io import read_resource_from_filepath
-from libecalc.presentation.yaml.yaml_entities import Resource
+from libecalc.presentation.yaml.yaml_entities import MemoryResource
 
 
-def _read_resources(directory: Path, resource_names: List[str]) -> Dict[str, Resource]:
+def _read_resources(directory: Path, resource_names: List[str]) -> Dict[str, MemoryResource]:
```

Comment:
replace this with file_resource_service?

Reply:
That would require a configuration, i.e. the yaml file parsed. We could look into using YamlModel here, with FileResourceService and FileConfigurationService. I would have to look into what YamlCase is used for in our tests.